packages/quantic/cypress/e2e/default-2/search-box/search-box.cypress.ts

```diff
@@ -0,0 +1,514 @@
+import {configure} from '../../../page-objects/configurator';
```

Comment:
do we have a playwright setup in quantic? If we are doing the transition from cypress to playwright do we not want to write the tests in playwright instead?

Reply:
Nope we don't have the setup yet in Quantic, we should do that before considering making tests in playwright plus this PR was made before the decision to start using playwright x)